lib/firefox/webdriver/builder.js

```diff
@@ -83,9 +83,13 @@ module.exports.configureBuilder = function(builder, baseDir, options) {
     }
   }
 
-  Object.keys(defaultFirefoxPreferences).forEach(function(pref) {
-    ffOptions.setPreference(pref, defaultFirefoxPreferences[pref]);
-  });
+  if (!firefoxConfig.noDefaultPrefs) {
+    Object.keys(defaultFirefoxPreferences).forEach(function(pref) {
+      ffOptions.setPreference(pref, defaultFirefoxPreferences[pref]);
+    });
+  } else {
+    log.info('Skip setting default preferences for Firefox');
```

Comment:
I saw that you added a log in the chrome PR so I've added one here as well for parity.

Reply:
Great!